{{cookiecutter.github_repository}}/provisioner/roles/common/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
 #  when: vm == 1
 
 # Ansible requires aptitude and python2 for various tasks
+- name: Set hostname
+  action: shell hostnamectl set-hostname {{ domain_name }}
```

Comment:
Is it required to have the hostname as domain name. can we maintain a separate variable for this.